packages/otelbin/src/components/monaco-editor/parseYaml.ts

```diff
@@ -250,3 +252,32 @@ export function selectConfigType(config: string) {
 		return config;
 	}
 }
+
+export function extractVariables(inputString: string): string[] {
+	const variableRegex = /\${([^}]+)}/g;
```

Comment:
The spec of the env var detection is here: https://github.com/dash0hq/otelbin/issues/156